apps/mobile/src/screens/edit-wheel-results/libs/components/scores-edit-form/scores-edit-form.tsx

```diff
@@ -0,0 +1,73 @@
+import { type BottomTabNavigationProp } from "@react-navigation/bottom-tabs";
+import React from "react";
+
+import {
+	Button,
+	MultipleSliderInput,
+	Text,
+	View,
+} from "~/libs/components/components";
+import { BottomTabScreenName } from "~/libs/enums/enums";
+import { useAppForm, useCallback, useNavigation } from "~/libs/hooks/hooks";
+import { globalStyles } from "~/libs/styles/styles";
+import { type BottomTabNavigationParameterList } from "~/libs/types/types";
+import {
+	type QuizScoresGetAllItemResponseDto,
+	type QuizScoresUpdateRequestDto,
+	updateScoresValidationSchema,
+} from "~/packages/quiz/quiz";
+
+import { SCORE_FORM_DEFAULT_VALUES } from "../../constants/constants";
+import { styles } from "./styles";
+
+type Properties = {
+	data: QuizScoresGetAllItemResponseDto[];
```

Comment:
```suggestion
	scores: QuizScoresGetAllItemResponseDto[];
```
Let's be more specific